app/models/spree_product_question/admin/main_menu/question_builder.rb

```diff
@@ -5,13 +5,19 @@ class QuestionBuilder
         include ::Spree::Core::Engine.routes.url_helpers
```

Comment:
@mausamp 
why is this inside folder `app/models/spree_product_question` and not `app/models/spree_product_qa`
name of gem is spree_product_qa, why would we create seperate folder than name of gem?